build-cmake/CMakeLists.txt

```diff
@@ -8,14 +8,23 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.10 FATAL_ERROR)
 cmake_policy(SET CMP0074 NEW)
 cmake_policy(SET CMP0091 NEW)
 cmake_policy(SET CMP0112 NEW)
+# TODO: Convert DART to CTEST
+cmake_policy(SET CMP0145 OLD)
 
 PROJECT(protobuf-c C CXX)
 
 if (MSVC AND NOT BUILD_SHARED_LIBS)
 	SET(Protobuf_USE_STATIC_LIBS ON)
 endif (MSVC AND NOT BUILD_SHARED_LIBS)
 
-FIND_PACKAGE(Protobuf REQUIRED)
+FIND_PACKAGE(Protobuf CONFIG)
+if(NOT Protobuf_FOUND)
+  MESSAGE(STATUS "Protobuf CMake config not found fallback to Cmake Module")
+  FIND_PACKAGE(Protobuf REQUIRED)
+else()
+  # Keep compatibility with FindProtobuf CMake module
+  set(PROTOBUF_PROTOC_EXECUTABLE $<TARGET_FILE:protobuf::protoc>)
+endif()
```

Comment:
Can you invert the sense of the `if(NOT Protobuf_FOUND)` conditional?

Basically this change: b92dabc1a56a0f8bc279b7bb6632da1aa98e0a94

Reply:
fixed